.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,27 @@
+### Description
+
```

Comment:
We should be using org wide templates not repo specific templates

Reply:
removed it